### PR TITLE
feat: add transparent variants

### PR DESCRIPTION
**Pull request by AdrienCos**
# Description

This PR add 4 new themes with a transparent background. The difference with the regular themes is simply that the background color is left empty. This allows terminals with a transparent background to work properly.

The theme files have been generated with Whiskers, by adding new entries to the matrix of the `.tera` file.

The inspiration comes from https://github.com/catppuccin/k9s which offers similar theme variants.

### CODE REVIEW

File: cava.tera [hunk 1]
```diff
@@ -2,13 +2,15 @@
 whiskers:
   version: 2.0.0
   matrix:
+    - variant: ["", "-transparent"]
```
Comment (uncenter) [inline cava.tera]:
```suggestion
    - variant: ["regular", "transparent"]
```

File: cava.tera [hunk 2]
```diff
@@ -2,13 +2,15 @@
...
     - flavor
-  filename: "themes/{{ flavor.identifier }}.cava"
+  filename: "themes/{{ flavor.identifier }}{{ variant }}.cava"
```
Comment (uncenter) [inline cava.tera]:
```suggestion
  filename: themes/{{ flavor.identifier }}{% if variant == "transparent" %}-{{ variant }}{% endif %}.cava
```

File: cava.tera [hunk 3]
```diff
@@ -2,13 +2,15 @@
...
 ---
 {%- set palette = flavor.colors -%}
 [color]
+{%- if variant == "" %}
```
Comment (uncenter) [inline cava.tera]:
```suggestion
{%- if variant == "regular" %}
```

**Review by uncenter — CHANGES_REQUESTED** [review]

**Comment by uncenter** [comment]
A little more verbose but it makes it more clear about the two types of variants.

File: cava.tera (hunk 2 above)
Comment (AdrienCos) [reply cava.tera]: I did try that before opening the PR, but got an error from Whiskers. It seems like using conditional blocks inside the `filename` field is not supported?

```sh
$ whiskers cava.tera
Error: Frontmatter is invalid

Caused by:
    Invalid YAML frontmatter (L6:62) : did not find expected key at line 6 column 62, while parsing a block mapping at line 2 column 3
```

```sh
$ whiskers --version
catppuccin-whiskers 2.1.1
```

Comment (uncenter) [reply cava.tera]: Ah, I think removing the quotes might work? I've updated the suggestion.

Comment (AdrienCos) [reply cava.tera]: It does, thanks! I've pushed the suggested changes

File: README.md [hunk 4]
```diff
@@ -18,8 +18,9 @@
 ## Usage
 
 1. Choose your flavour.
-2. Copy the contents of `themes/<flavour>.cava` into your Cava config file (usually located at: `$HOME/.config/cava/`), replacing the existing gradient settings.
-3. Reload cava if it was already playing.
+2. Choose your background variant: opaque or transparent.
+3. Copy the contents of `themes/<flavour>.cava` or `themes/<flavour>-transparent.cava` into your Cava config file (usually located at: `$HOME/.config/cava/`), replacing the existing gradient settings.
```
Comment (uncenter) [inline README.md]:
```suggestion
3. Copy the contents of `themes/<flavor>.cava` or `themes/<flavor>-transparent.cava` into your Cava config file (usually located at: `$HOME/.config/cava/`), replacing the existing gradient settings.
```
Minor spelling nit!

Comment (AdrienCos) [reply README.md]: Good catch

I've also applied the same fix to point `1.` of the README

**Review by uncenter — APPROVED** [review]

File: cava.tera (hunk 2 above)
Comment (AdrienCos) [reply cava.tera]: Oh btw, do you think this quotes thing can be considered an bug in Whiskers, and should I report it ? Or is it supposed to error out like it does?

**Review by sgoudham — APPROVED** [review]
Works locally, thanks for the PR!

**Pull request merged**